main/clamav/APKBUILD

```diff
@@ -53,6 +53,11 @@ builddir="$srcdir/$pkgname-$pkgver"
 #     - CVE-2017-16932
 #     - CVE-2018-0360
 #     - CVE-2018-0361
+#   0.100.2-r0:
+#     - CVE-2018-15378
+#     - CVE-2018-14680
+#     - CVE-2018-14681
+#     - CVE-2018-14682
```

Comment:
Makes sense to mention CVEs in commit message

Reply:
Done